### ISSUE TITLE
Extremities cold to touch

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10280]
**Preferred term label:**
Extremities cold to touch

**Synonyms**
Peripheral vasoconstriction; cool or cold arms or legs

**Definition (free text, please give PubMed ID)**
This is different from [Raynaud phenenomenon](https://hpo.jax.org/app/browse/term/HP:0030880), with no discoloration and with equal expression in arms and legs 

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0025323

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Cardiogenic shock
Blood loss, acute (hours)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I do not think that Abnormal arterial physiology HP:0025323 is the proper parent term, because this can be nornmal physiology (e.g., following a long period of being outside in cold temperatures). 
I think this is better as [Constitutional symptom HP:0025142](https://hpo.jax.org/app/browse/term/HP:0025142), which is non-commital as to where the symptom comes from. Thoughts?

**Comment by MickeySegal** [comment]
I had just picked that parent term to make clear the distinction from Raynaud, but a different parent would be fine with me.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000500
- label: Extremities cold to touch
- parents: [Constitutional symptom (HP:0025142)](https://hpo.jax.org/app/browse/term/HP:0025142)
- pmid: PMID:34645109
- def: Cool or cold arms or legs related to peripheral vasoconstriction, the narrowing of blood vessels resulting from the contraction of muscular walls in large arteries and the small arterioles of skin and most internal organs. Following exposure to low ambient temperatures, vasoconstriction decreases heat dissipation from skin surfaces, thus preventing life-threatening hypothermia.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10280
- synonyms: Peripheral vasoconstriction(NONE)
